app/src/main/java/org/oppia/app/profile/AdminAuthEnum.kt

```diff
@@ -0,0 +1,8 @@
+package org.oppia.app.profile
+
+import org.oppia.app.R
+
+/** Represents different activities from which AdminPinActivity can be opened*/
```

Comment:
/** Represents different activities from which AdminPinActivity can be opened. */

Reply:
Done